### ISSUE TITLE
Missing step in preparation listing

### ISSUE THREAD

**Issue by ghost** [gtbjj/x205ta#4]
Hi,

First of all thanks for this tutorial, it is very useful.

It must be one month since I'm trying to install Arch on my x205ta and I've not yet been successful into this task.

The more I try, the more I think that the step where I make a mistake is writing the iso on the thumb drive.

And I guess that there is a forgotten step — well at least a not documented into this tutorial — where I get lost into the Arch documentation wiki when I need to remaster the arch iso.

I just add the bootia32.efi file on the unpacked archiso where it is writable, and that's fine. But the direct next step is writing the archx205ta.iso to the thumb drive with Rufus (bonus question : could I dd it on the drive as far as it is a fat32 with boot flag active one ?), but at this step — and if I don't miss anything — there is not yet a archx205ta.iso file anywhere, just a folder somewhere in my /home with all the iso files plus one.

So what am I missing? (and if I'm missing somewhat, how can I solve this issue ?).

Thanks a lot!

**Comment by savagezen** [comment]
My apologies for the incongruence.  I had updated some things and forgotten to change others.  The [Preparation](https://github.com/gtbjj/x205ta) and [Installation](https://github.com/gtbjj/x205ta/blob/master/INSTALL.md) pages have been updated.

The missing step for you was to generate the ISO via `genisoimage`.  The instructions have been updated.  It's also possible to write the stock ISO image to the USB via Rufus, open the drive in Windows, and drag / drop the bootia32.efi file to its needed location and boot the USB per usual without regenerating the image.
